### PR TITLE
API1800 support Logical Interconnects

### PR DESCRIPTION
**Pull request by yuvirani**
### Description
API1800 support Logical Interconnects

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: lib/oneview-sdk/resource/api1800/c7000/logical_interconnect.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+module OneviewSDK
+  module API1800
+    module C7000
+      # Logical interconnect resource implementation for API1800 C7000
+      class LogicalInterconnect < OneviewSDK::API1600::C7000::LogicalInterconnect
+        # Gets igmpSettings of logical interconnect
+        def get_igmp_settings
+          ensure_client && ensure_uri
+          response = @client.rest_get("#{@data['uri']}/igmpSettings")
```
Comment (VenkateshRavula) [inline lib/oneview-sdk/resource/api1800/c7000/logical_interconnect.rb:22]: why "#" before {@data['uri']}

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,7 +13,9 @@ This release extends support to OneView Rest API version 1800 (OneView v5.30) fo
 - Hypervisor Cluster Profile
 - Hypervisor Manager
 - Interconnects
+- Interconnect Types
```
Comment (VenkateshRavula) [inline CHANGELOG.md:16]: why did u add this resource when there is no lib files related to interconnect types.?

Comment (yuvirani) [reply CHANGELOG.md:16]: its covered in Interconnects resource itself. We had it separately in previous release as well. Hence followed the same

File: lib/oneview-sdk/resource/api1800/c7000/logical_interconnect.rb (hunk 1 above)
Comment (yuvirani) [reply lib/oneview-sdk/resource/api1800/c7000/logical_interconnect.rb:22]: This is just to interpret whatever given inside double quotes which is basically equals to rest_get(@data['uri'] + '/igmpSettings')

Comment (VenkateshRavula) [reply lib/oneview-sdk/resource/api1800/c7000/logical_interconnect.rb:22]: ok

File: endpoints-support.md [hunk 3]
```diff
@@ -189,34 +189,36 @@ OneviewSDK::Datacenter.find_by(@client, width: 11000).map(&:remove)
...
-|<sub>/rest/logical-interconnects/compliance</sub>                                        | PUT      | :heavy_multiplication_x:       | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: |
-|<sub>/rest/logical-interconnects/{id}</sub>                                              | PATCH     | :white_check_mark:   | :white_check_mark:   | :heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   | :heavy_minus_sign:   |
+|<sub>/rest/logical-interconnects</sub>                                                   | GET       | :white_check_mark:   | :white_check_mark:   |  :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/logical-interconnects/locations/interconnects</sub>                           | POST      | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:   |
```
Comment (chebroluharika) [inline endpoints-support.md:193]: are we supporting this either in C7000 or Synergy?

Comment (yuvirani) [reply endpoints-support.md]: Done

**Pull request merged**